src/v2/view-builder/components/AuthenticatorVerifyFooter.js

```diff
@@ -0,0 +1,8 @@
+import BaseFooter from '../internals/BaseFooter';
```

Comment:
Why is this called AuthenticatorVerifyFooter? and not something more generic?

Reply:
Aren't only page for authenticator verify having `use something else` link?
Otherwise we could rename it to more generic.

Reply:
I see ok, that makes sense